src/test/resources/org/jenkinsci/plugins/ewm/casc/configuration-as-code.yaml

```diff
@@ -0,0 +1,54 @@
+jenkins:
+  systemMessage: "Jenkins configured automatically by Jenkins Configuration as Code Plugin"
+  numExecutors: 5
+  scmCheckoutRetryCount: 2
+  mode: NORMAL
+  nodeProperties:
+    - exwsNodeConfigurationDiskPools:
+        nodeDiskPools:
+          - diskPoolRefId: "master-node-id"
+            nodeDisks:
+              - diskRefId: "master-node-disk"
+                nodeMountPoint: "/tmp/master-node"
+
+  nodes:
+    - permanent:
+        name: "static-agent"
+        remoteFS: "/home/jenkins"
+        nodeProperties:
+          - exwsNodeConfigurationDiskPools:
+              nodeDiskPools:
+                - diskPoolRefId: "localhostdiskpool"
+                  nodeDisks:
+                    - diskRefId: "localdisk"
+                      nodeMountPoint: "/tmp/localdisk"
+  slaveAgentPort: 50000
+  agentProtocols:
+    - "jnlp2"
+
+unclassified:
+  exwsGlobalConfigurationDiskPools:
+    diskPools:
+      - diskPoolId: "diskpool1"
+        disks:
+          - diskId: "disk1"
+            displayName: "disk one display name"
+            diskInfo: "noDiskInfo"
+            masterMountPoint: "/tmp"
+        displayName: "diskpool1 display name"
+        restriction: "any"
+        strategy: "mostUsableSpace"
+  exwsGlobalConfigurationTemplates:
+    templates:
```

Comment:
I would like to see a list of templates in the test to make sure the loops are operating properly.

Reply:
I will add additional templates and accompanying unit tests for this.

Reply:
https://github.com/imaffe/external-workspace-manager-plugin/pull/9

Reply:
The underlying PR was merged, resolving this conversation.